.travis.yml

```diff
@@ -18,9 +18,12 @@ script:
   - 'if [ "${TEST-}" = true ]; then npm run tests-only ; fi'
   - 'if [ -n "${KARMA-}" ]; then npm run tests-karma ; fi'
   - 'if [ -n "${COVERAGE-}" ] && [ "${TRAVIS_BRANCH-}" = "master" ]; then npm run cover; cat ./coverage/lcov.info | ./node_modules/.bin/coveralls ; fi'
+  - 'if [ -n "${HAPPO-}" ]; then npm install --no-save happo-plugin-puppeteer && INSTALL_CMD="npm install && npm install --no-save happo-plugin-puppeteer" npm run happo-ci ; fi'
```

Comment:
slightly more concise? `INSTALL_CMD="npm install && npm install --no-save happo-plugin-puppeteer" && $INSTALL_CMD && npm run happo-ci`?

Reply:
Done!

Reply:
I had to roll that back. I tried multiple versions of this but I couldn't get it right.